src/cython_cmake/cmake/UseCython.cmake

```
@@ -94,26 +94,29 @@ function(Cython_compile_pyx)
 
   # Set target language
   if(NOT CYTHON_LANGUAGE)
-    get_property(_languages GLOBAL PROPERTY ENABLED_LANGUAGES)
+    get_property(_langauges GLOBAL PROPERTY ENABLED_LANGUAGES)
```

```suggestion
get_property(_languages GLOBAL PROPERTY ENABLED_LANGUAGES)
```

I believe original spelling was correct